docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+
+##### Video Mappings
+
+In order that a NMOS device that provides IS-04 Receivers can work with EDID type video devices the device:
+
+- MUST create one IS-04 Video Receiver for each of the displays associated with the IS-04 Device
+
+- MUST set the `format` element to the value `urn:x-nmos:format:video`  
+
+- MUST associate with each IS-04 Receiver a `contraints_set` resource in the IS-04 Receiver's `caps` resource  as described in [BCP-004-1]
+
+- MUST map at least one of the video modes present by the display device's EDID into the created `contraint_set` with the following elements set:
+
+  - `urn:x-nmos:cap:meta:enabled` set to `true`.  
+
+  - `urn:x-nmos:cap:format:grain_rate` set to the refresh rate as taken from the EDID's preferred timing mode for the supported video format. 
+
+  - `urn:x-nmos:cap:format:frame_width` set to the video frame width as taken from the EDID preferred timing mode.
+
+  - `urn:x-nmos:cap:format:frame_height` set to the video frame height as taken from the EDID preferred timing mode.
+
+  - `urn:x-nmos:cap:format:interlace_mode` set to the interlace mode as taken from the EDID standart time block for this supported video mode.
+
+  - `urn:x-nmos:cap:format:colorspace` set to the colorspace indicated in the EDID Feature Support byte.
+
+  - `urn:x-nmos:cap:format:color_sampling` set to the set to the color sampling indicated in the EDID Feature Support byte.
+
+  - `urn:x-nmos:cap:format:component_depth` set to the color bit depth field of the EDID's Video Input definition.
+
+- MUST map the following elements if present in the EDID
+
+  - `urn:x-nmos:cap:format:transfer_characteristic` set to the HDR capabilities of the monitor if the EDID contains the optional HDR metadata block.
+
+  - `urn:x-nmos:cap:meta:preference` 
+    - set to 100 if the video mode is a preferred mode.  
+    - set to -100 if the video mode is a non-native mode.  
+    - set to 0 all others  
+
+- MUST not add a `contraints_set` for any video mode the device cannot support due to implementation limitations even though the display supports the mode.
+
+- MUST add a grouping tag to the created video receiver with tag `name` and `role` set to indicate a group comprised of the video receiver and audio receiver associated with the display.
+
+- SHOULD provide a `contraint_sets` entry for each of the video modes supported by the display as provided by the EDID as an individual `contraint_set`on the receiver with parameters set as described above if the mode is supported by the NMOS device.
+
+##### Audio Mappings
+
+In order that a NMOS device that provides Audio Receivers can work with EDID type audio devices the device:
+
+- MUST create one IS-04 Audio Receiver for each of the displays associated with the IS-04 Device that support audio.
+
+- MUST set the `format` element to the value `urn:x-nmos:format:audio`  
```

Comment:
Should these definitions be specific to a Receiver of a specific format? In NMOS, a device is not specific to a format so I suspect the requirements need to be applied on a different resource